python/paddle/distributed/auto_parallel/static/pir_pass.py

```diff
@@ -1048,20 +1042,40 @@ def complete_chunk_id(dist_program, startup_program, pipeline_strategy):
     # Step3: Get op index boundary, pp_stage, chunk_id, struct_names of each segment
     seg_pp_stages = [i % pp_degree for i in range(num_chunks)]
     seg_chunk_ids = [i // pp_degree for i in range(num_chunks)]
-    seg_layer_num = [0] * num_chunks
-    for j in range(0, len(seg_struct_names)):
-        i = j % num_chunks
-        seg_layer_num[i] = seg_layer_num[i] + 1
     seg_parts = [0]
-
+    last_struct_name = None
+    user_layer_to_stage_id = (
+        []
+    )  # User intent - a list corresponding to the model layer and the device. The key of the list is the number of the layer, and the value is the corresponding pp_stage.
```

Comment:
```suggestion
    stage_ids=[]
```
stage_ids[i]表示第i层分配的stage编号，这样是否更简洁一些？

Reply:
Done